.github/CODEOWNERS

```diff
@@ -256,6 +256,14 @@
 
 /theories/Vectors/        @herbelin
 
+########## Dune  ##########
+
+/Makefile.dune         @ejgallego
+/tools/coq_dune*       @ejgallego
+/dune*                 @ejgallego
+/*.opam                @ejgallego
```

Comment:
This excludes `ide/coqide.opam` doesn't it? Is it on purpose?

Reply:
Nope, it was an unintentional omission [`opam` files are redundant with `dune-project` and they will hopefully go away soon (maybe dune 1.2)]

Reply:
Fixed, note also that I've left the `dune` files as owned by the particular maintainer of each library; IMHO that is OK, but note that `dune` files may go away soon too. Dune 1.1 supports recursive libraries so would be add all the ml parts into a directory `core/kernel`, `core/pretyping`, etc... we could use a single `dune` file for `core` and have libraries automatically named `Coq.Kernel` etc...

In fact upstream has stated that the goal is to have a single `dune-project` file and have build rules automatically generated by convention. So we are far from this but sometime time will come.